app/core/downloader/episode.py

```
@@ -1,5 +1,6 @@
 from typing import Optional
 from urllib.parse import urlparse
+from typing import Optional
```

Ruff (0.14.14)

[error] Redefinition of unused `Optional` from line 1: `Optional` redefined here

Remove definition: `Optional`

(F811)